scripts/release.js

```diff
@@ -68,6 +73,14 @@ async function upload(meta) {
         mime.getType(filePath.replace('.gz', '')) || 'application/javascript',
     }
 
+    const shadowOptions = {
+      Bucket: shadowBucket,
+      Key: path.join(`analytics-next`, meta.branch, meta.sha, f),
+      Body: await fs.readFile(filePath),
+      ContentType:
+        mime.getType(filePath.replace('.gz', '')) || 'application/javascript',
+    }
```

Comment:
Looks like this can be simplified and remove an IO operation!
```suggestion
    const shadowOptions = {
      ...options,
      Bucket: shadowBucket,
    }
```